overview.md

```diff
@@ -206,3 +221,8 @@ variables:
     command: preview
     stack: $(StackName)
 ```
+
+### `Error: TF401027: You need the Git 'PullRequestContribute' permission to perform this action.`
+
```

Comment:
This troubleshooting error is related to this new feature. Users who do not grant the required permission to the build service's user will see this error in their pipeline if they try to use the PR comment feature without it.

Reply:
Brilliant!